### PR TITLE
Updated the sponsors mentioned in the read-me and website

### PR DESCRIPTION
**Pull request by dennisdoomen**
* Updated the sponsors mentioned in the read-me and website
* Also removed the `gemfile.lock` so make `bundle install` to work.

### CODE REVIEW

File: docs/Gemfile.lock
Comment (jnyrup) [inline docs/Gemfile.lock:1]: Is this removal intentional?

File: README.md [hunk 1]
```diff
@@ -32,14 +32,13 @@ This is a special set of tests that use the [Verify](https://github.com/VerifyTe
 If you've verified the changes and decided they are valid, you can accept them  using `AcceptApiChanges.ps1` or `AcceptApiChanges.sh`. Alternatively, you can use the [Verify Support](https://plugins.jetbrains.com/plugin/17240-verify-support) plug-in to compare the changes and accept them right from inside Rider. See also the [Contribution Guidelines](CONTRIBUTING.md).
 
 # Powered By
-<a href="https://aws.amazon.com/"><img src="docs/assets/images/aws.png" style="width:100px"/></a>&nbsp;
```
Comment (jnyrup) [inline README.md:35]: Do we use the aws.png elsewhere?

File: docs/Gemfile.lock
Comment (dennisdoomen) [reply docs/Gemfile.lock:1]: Yeah, without it I couldn't get `bundle install` to complete. I'll mention that in the description.

File: README.md (hunk 1 above)
Comment (dennisdoomen) [reply README.md:35]: Probably not. I'll remove it.

File: docs/Gemfile.lock
Comment (jnyrup) [reply docs/Gemfile.lock:1]: What error did you experience?
I did this:
* checked out this PR
* restored Gemfile.lock from master
* ran `bundle install` from an elevated powershell
* ran `bundle exec jekyll serve`

Comment (dennisdoomen) [reply docs/Gemfile.lock:1]: It hangs at installing `ffi`

image
